hnn_core/gui/gui.py

```diff
@@ -536,91 +842,97 @@ def on_upload_change(change, sliders, params, tstop, tstep, log_out, variables,
                 drive_boxes, tstop)
 
 
+def init_network_from_widgets(params, tstep, tstop, variables, drive_widgets):
+    """Construct network and add drives."""
```

Comment:
Document all public functions and methods!

Reply:
I think I'll turn this into a private one... as no other part of hnn-core should call this